lib/cuckoo/common/integrations/pdfminer.py

```diff
@@ -0,0 +1,64 @@
+import logging
+from typing import Any
+from typing import Dict
+from typing import Iterable
+from typing import Set
+from typing import Union
+
+try:
+    from pdfminer import pdfparser
+    from pdfminer import pdfdocument
+    from pdfminer import pdftypes
+
+    HAVE_PDFMINER = True
+except ImportError:
+    HAVE_PDFMINER = False
+
+log = logging.getLogger(__name__)
+
+
+def _search_for_url(obj: Union[pdftypes.PDFStream, dict, list]) -> Iterable[str]:
```

Comment:
what if we move out `pdftypes.PDFStream ` from Union? to not fail on test if not installed as not all users will have it